week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -63,9 +63,9 @@ test(
 );
 test(
   "sortArray function works",
-  sortArray() === ["a", "c", "e", "f", "n", "z"]
+  sortArray() === [ 'a', 'c', 'e', 'f', 'n', 'z' ].toString()
 );
-test("first5 function works", first5() === [1, 2, 3, 4, 5]);
+test("first5 function works", first5() === [1, 2, 3, 4, 5].toString());
 
 test(
   "get3rdIndex function works - case 1",
```

Comment:
You run `toString()` on a lot of arrays here - can you tell me why and what you think it's doing?

Reply:
To compare arrays.  It turns the array into a string